src/components/TechnicalInfoContent/TechnicalInfoContent.js

```diff
@@ -58,15 +77,24 @@ export const TechnicalInfoContent = () => {
       <Grid item>
         <Typography gutterBottom={false}>{t('genericcomponent.dialog.technicalInfo.details', 'Details')}</Typography>
         <List dense disablePadding>
-          {technicalInformation
-            .filter((infoItem) => infoItem.content != null)
-            .map((infoItem, index) => (
-              <ListItem
-                key={index}
-                sx={{ display: 'list-item', listStyleType: 'disc', listStylePosition: 'inside' }}
-                dense
-              >{`${infoItem.label}: ${infoItem.content}`}</ListItem>
-            ))}
+          {infoToDisplay.map((infoItem, index) => (
+            <ListItem
+              key={index}
+              sx={{ display: 'list-item', listStyleType: 'disc', listStylePosition: 'inside' }}
+              dense
+            >
+              <Typography component="span">{infoItem.label + ' '}</Typography>
+              {infoItem.content ? (
+                <Typography component="span" sx={{ fontWeight: 'bold' }}>
+                  {infoItem.content}
+                </Typography>
+              ) : (
+                <Typography component="span" sx={{ fontWeight: 'bold' }}>
+                  <i>{t('genericcomponent.dialog.technicalInfo.notAvailable')}</i> {'(' + infoItem.content + ')'}
+                </Typography>
+              )}
```

Comment:
In the second part of the ternary, we're using `infoItem.content` even though it's `null` or `undefined`, is it done on purpose?

Also, I think parts of the code can be factorized for the `ListItem` content:
```
      <Typography component="span">{infoItem.label + ' '}</Typography>
      <Typography component="span" sx={{ fontWeight: 'bold' }}>
        {infoItem.content ?? (
          <>
            <i>{t('genericcomponent.dialog.technicalInfo.notAvailable')}</i>
            {'(' + infoItem.content + ')'})
          </>
        )}
      </Typography>
```

Reply:
I think the purpose is especially to display _infoItem_ as _null_ or as _undefined_